web/src/locales/hi.json

```diff
@@ -0,0 +1,1894 @@
+        "US-NW-WAUW": {
+            "countryName": "USA",
+            "displayName": "WAPA Upper Great Plains",
+            "zoneName": "Western Area Power Administration - Upper Great Plains West"
+        },
+        "US-NY-NYIS": {
+            "countryName": "USA",
+            "zoneName": "New York ISO",
+            "seoZoneName": "New York ISO - NYISO"
+        },
+        "US-SE-SEPA": {
+            "countryName": "USA",
+            "zoneName": "Southeastern Power Administration"
+        },
+        "US-SE-SOCO": {
+            "countryName": "USA",
+            "zoneName": "Southern Company Services"
+        },
+        "US-SW-AZPS": {
+            "countryName": "USA",
+            "zoneName": "Arizona Public Service Company"
+        },
+        "US-SW-EPE": {
+            "countryName": "USA",
+            "zoneName": "El Paso Electric Company"
+        },
+        "US-SW-PNM": {
+            "countryName": "USA",
+            "zoneName": "Public Service Company of New Mexico"
+        },
+        "US-SW-SRP": {
+            "countryName": "USA",
+            "zoneName": "Salt River Project"
+        },
+        "US-SW-TEPC": {
+            "countryName": "USA",
+            "zoneName": "Tucson Electric Power Company"
+        },
+        "US-SW-WALC": {
+            "countryName": "USA",
+            "displayName": "WAPA Desert Southwest",
+            "zoneName": "Western Area Power Administration - Desert Southwest Region"
+        },
+        "US-TEN-TVA": {
+            "countryName": "USA",
+            "displayName": "TVA",
+            "zoneName": "Tennessee Valley Authority",
+            "seoZoneName": "Tennessee Valley Authority - TVA"
+        },
+        "US-TEX-ERCO": {
+            "countryName": "USA",
+            "displayName": "ERCOT",
+            "zoneName": "Electric Reliability Council of Texas",
+            "seoZoneName": "Electric Reliability Council of Texas - ERCOT"
+        },
+        "UY": {
+            "zoneName": "Uruguay"
+        },
+        "UZ": {
+            "zoneName": "Uzbekistan"
+        },
+        "VC": {
+            "zoneName": "Saint Vincent and the Grenadines"
+        },
+        "VE": {
+            "zoneName": "Venezuela"
+        },
+        "VI": {
+            "countryName": "USA",
+            "zoneName": "Virgin Islands"
+        },
+        "VN": {
+            "zoneName": "Vietnam"
+        },
+        "VU": {
+            "zoneName": "Vanuatu"
+        },
+        "WS": {
+            "zoneName": "Samoa"
+        },
+        "XK": {
+            "zoneName": "Kosovo"
+        },
+        "XX": {
+            "zoneName": "Northern Cyprus"
+        },
+        "YE": {
+            "zoneName": "Yemen"
+        },
+        "YT": {
+            "zoneName": "Mayotte"
+        },
+        "ZA": {
+            "zoneName": "South Africa"
+        },
+        "ZM": {
+            "zoneName": "Zambia"
+        },
+        "ZW": {
+            "zoneName": "Zimbabwe"
+        }

```

Comment:
I assume these have translated versions of at least the countries, why are they not added?

If temporary these keys should not be added at all so it can fall back to the english translation.